pkg/oauth/util/discovery_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func TestGet(t *testing.T) {
-	actual := Get("https://localhost:8443", "https://localhost:8443/oauth/authorize", "https://localhost:8443/oauth/token")
+	actual := GetOauthMetadata("https://localhost:8443")
```

Comment:
nvm, this is consistent with the containing file and the whole code base is really inconsistent on this anyway.